webapp/app/css/treeherder.css

```diff
@@ -790,6 +790,7 @@ ul.failure-summary-list li .btn-xs {
 
 .failure-summary-line-empty {
     padding: 2px 4px 0px;
+    font-size: 12px;
```

Comment:
Here we bump up the size of the failure summary line to a 'standard' of 12px also, so they are consistent.